base/Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM debian:sid
 MAINTAINER Zero Cho "http://itsze.ro"
 
-RUN apt-get update
-RUN apt-get install -y scala git curl openjdk-7-jdk netcat
+RUN apt-get update && apt-get install -y \
+  curl \
+  openjdk-8-jre-headless \
```

Comment:
How about using `openjdk-8-jre` as the base image? It also includes `curl` (but probably not `netcat`, which, are we really using it?)

See https://github.com/docker-library/java/blob/0f82de10e4aaa8b78d7f79cf725fd1fad1dc85e0/openjdk-8-jre/Dockerfile for what is does exactly.